webapp/src/pages/createApplicationForm/components/util.js

```diff
@@ -222,3 +233,240 @@ export const handleMove = ({
     setAnimation && setAnimation(true);
   }
 }
+
+export const Dependency = ({
+  options, handlequestionDependency, inputs, lang,
+  dependentQuestion, handleDependencyChange, t
+}) => {
+  const dependency = options.find(o => o.value === inputs.depends_on_question_id);
+  
+  return (
+    <div className='dependency-wrapper'>
+      <ReactSelect
+        options={options}
+        placeholder={t('Depends on question')}
+        onChange={e => handlequestionDependency(e)}
+        value={dependency ? dependency : null}
+        className='select-dependency'
+        styles={{
+          control: (base, state) => ({
+            ...base,
+            boxShadow: "none",
+            border: "none",
+            transition: state.isFocused 
+              && 'color,background-color 1.5s ease-out',
+            background: state.isFocused
+              && 'lightgray',
+            color: '#fff'
+          }),
+          option: (base, state) => ({
+            ...base,
+            backgroundColor: state.isFocused
+              && "lightgray",
+            color: state.isFocused && "#fff"
+          })
+        }}
+        menuPlacement="auto"
+      />
+      {dependentQuestion && dependentQuestion.type
+        && dependentQuestion.type.value === 'single-choice'
+        && (
+        <div className='dependency-radio-wrapper'>
+          <input
+            id='single_choice'
+            type='checkbox'
+            className='single-choice-check'
+            checked={inputs.show_for_values}
+            onChange={handleDependencyChange('single')}
+            />
+          <label htmlFor='single_choice'>
+            {inputs.show_for_values ? 'True' : 'False'}
```

Comment:
I found the checkbox which toggles between True and False when the dependency question is type "single-choice" to be a bit confusing... Can you rather change this to two radio buttons with "True" and "False" please?